airbyte-integrations/connectors/destination-google-sheets/integration_tests/integration_test.py

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
```

Comment:
is there a reason that all tests are in integration_tests and we haven't separated unit tests into a separate folder (like with sources)?

Reply:
Yes, the reason is that unit_tests are not running during the `/test` for python destinations, probably due to missing `gradlew` scenario, haven't figure it out yet. The process of `integrationTests` performs the `customIntegrationTests` and skips the `unit_tests` part, unlike the source connectors. Another reason is we need to hit the test_coverage up to 90% all in one place, so it's decided to put all the tests in integration_tests folder.